src/azure-cli-core/azure/cli/core/azclierror.py

```diff
@@ -18,6 +18,8 @@ class AzCLIErrorType(Enum):
     # userfaults
     CommandNotFoundError = 'CommandNotFoundError'
     ArgumentParseError = 'ArgumentParseError'
+    UsageError = 'UsageError'
```

Comment:
what's usage error means normally? Invalid argument value? It's little too ambiguous, thoughts?